scripts/update-bcd.test.ts

```diff
@@ -961,38 +1000,109 @@ describe("BCD updater", () => {
             ["81", true],
             ["82", false],
           ]),
-          {
-            version_added: "≤80",
-          },
+          [
+            {
+              version_added: "≤80",
+            },
+          ],
         ),
         "detects possible support removal",
       );
+    });
+
+    it("detects updates across multiple default statements", () => {
+      assert.isFalse(
+        hasSupportUpdates(
+          new Map([
+            ["10", true],
+            ["29", false],
+            ["30", true],
+          ]),
+          [
+            {
+              version_added: "30",
+            },
+            {
+              version_removed: "20",
+              version_added: "≤10",
```

Comment:
I'd suggest not using ≤ ranges in most tests, and have dedicated tests for that. So here:

```suggestion
              version_added: "10",
```